packages/slice-machine/components/Simulator/components/Toolbar/ScreensizeInput/index.tsx

```diff
@@ -0,0 +1,94 @@
+import { ScreenDimensions } from "@lib/models/common/Screenshots";
+import { Flex, Input, Label, ThemeUIStyleObject } from "theme-ui";
+
+export enum ScreenSizeOptions {
+  DESKTOP = "Desktop",
+  TABLET = "Tablet",
+  MOBILE = "Mobile",
+  CUSTOM = "Custom",
+}
+
+export const ScreenSizes: Record<string, ScreenDimensions> = {
+  [ScreenSizeOptions.DESKTOP]: {
+    width: 1200,
+    height: 800,
+  },
+  [ScreenSizeOptions.TABLET]: {
+    width: 1024,
+    height: 800,
+  },
+  [ScreenSizeOptions.MOBILE]: {
+    width: 400,
+    height: 850,
+  },
+};
+
+type ScreensizeInputProps = {
+  label: "H" | "W";
+  startValue: number;
+  onChange: (sizeEvent: React.ChangeEvent<HTMLInputElement>) => void;
+  isActive: boolean;
+  sx?: ThemeUIStyleObject;
+};
+
+export const ScreensizeInput: React.FC<ScreensizeInputProps> = ({
```

Comment:
I've asked Benjamin about the behavior of the inputs, he told me that the acceptance criteria where wrong.

The input should never be disabled and the dropdown selected option should change according to the selected values in the Input (something known, or unknown = Custom).

Reply:
I've updated this now

Reply:
thanks 👍  works well.